docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -35,56 +35,100 @@ Every cluster has a statistics tree rooted at *cluster.<name>.* with the followi
   upstream_cx_connect_fail, Counter, Total connection failures
   upstream_cx_connect_timeout, Counter, Total connection connect timeouts
   upstream_cx_idle_timeout, Counter, Total connection idle timeouts
-  upstream_cx_connect_attempts_exceeded, Counter, Total consecutive connection failures exceeding configured connection attempts
-  upstream_cx_overflow, Counter, Total times that the cluster's connection circuit breaker overflowed
+  upstream_cx_connect_attempts_exceeded, Counter, "| Total consecutive connection failures
+  | exceeding configured connection attempts"
```

Comment:
Is this a known trick for Sphinx? If so, could you add a reference to something canonical. Otherwise, can you comment on how stable this will be across Sphinx or theme changes? I like the idea and think we should do this, but would be good to understand the magic a bit.

Reply:
I could not find any thing related to this Sphinx docs, but found this on stackoverflow https://stackoverflow.com/questions/34194728/csv-table-formatting-in-python-docstrings-sphinx-multiple-lines-in-one-cell

Reply:
BTW, `|` in csv column seems to be a standard Sphinx to get a new line in line blocks http://rest-sphinx-memo.readthedocs.io/en/latest/ReST.html#line-blocks.  The only thing that is magic here is alignment of those lines using css.

Reply:
Did you manually go through and break all of these? I'm just trying to understand what is expected in the future? If it does auto break, are these continuations really necessary? Or was the done just for code readability?

Reply:
Yes. I have manually changed all of them. It is necessary to break the line the way I did if it has to come in multiple lines in that table. Should we document it some where?

Reply:
My fear is that inevitably someone is going to forget and it's a little weird. There isn't any way to have it auto break? It seems like it should be possible somehow?

Reply:
AFAIK, auto break is not possible. I will search again and let you know though.

Reply:
@mattklein123  checked again, could not find a way to auto break. One thing though, if they forget, that particular row needs to be scrolled, rest of the table looks fine. If you think, it will be a maintenance overhead and people may forget , I am ok to close this PR. let me know.

Reply:
We could always have {check,fix}_format do the breaking at the expense of some more seedy regex fu.

Reply:
Yeah I dunno. I'm kind of on the fence here. @envoyproxy/maintainers any opinions?

Reply:
I would object to this kind of manual (hard) line break purely for styles. I feel this should be handled by CSS. With manually broken lines it still doesn't show well when you zoom in / out on browser. Also it might be bad for other accessibility function like screen readers.